types/tektelic/edoctorV1/default.schema.json

```
@@ -0,0 +1,62 @@
+{
+  "$id": "https://akenza.io/tektelic/edoctorV1/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "topic": "default",
+  "processingType": "uplink_decoder",
+  "properties": {
+    "skinTemperature": {
+      "title": "Skin temperature",
+      "unit": "°C",
+      "type": "number",
+      "description": "Skin temperature"
```

Should we be a bit more precise here?

```suggestion
      "description": "Skin temperature in degrees Celcius"
```